src/Exception/Strategy/StrategyManager.php

```diff
@@ -171,7 +171,7 @@ public function getBlockFilter(BlockInterface $block): FilterInterface
      *
      * @throws \RuntimeException
      */
-    private function getFilterService(string $name): object
+    private function getFilterService(string $name): ?object
```

Comment:
I would keep the `object` return and throw an exception if the filter is null / undefined

Reply:
Currently the check is done in `getBlockFilter`.

It could be smarter to move the check inside this function.